serializers/index.js

```diff
@@ -38,7 +38,7 @@ var common_serializers = {
   },
   res: function (res) {
     if (!res) return {};
-    var headers = res.headers || {};
+    var headers = (typeof res.getHeaders === 'function') ? res.getHeaders() : headers || {};
```

Comment:
Isn't this missing the `res.` in front of `headers`?

```suggestion
    var headers = (typeof res.getHeaders === 'function') ? res.getHeaders() : res.headers || {};
```

Reply:
+1

Reply:
good catch 🙇